hyp3_rtc_gamma/rtc_sentinel.py

```diff
@@ -595,7 +595,7 @@ def rtc_sentinel_gamma(in_file,
                 browse_res, dem, terms, par=par, orbit_file=orbit_file)
 
     if include_scattering_area:
-        create_area_geotiff(f'geo_{pol}/image_1.pix', f'geo_{pol}/image_1.map_to_rdc', f'{out_name}.{pol}.mgrd.par',
+        create_area_geotiff(f'geo_{pol}/image_gamma0_1.pix', f'geo_{pol}/image_1.map_to_rdc', f'{out_name}.{pol}.mgrd.par',
```

Comment:
Is this change related to `mk_geo_radcal2`?

Reply:
Ah, I think I see it, we'll likely need to reference our radiometry flag to know which image to pull here.  It looks like the patch you posted is only the additional changes, not including the pre-existing patch to create the -g flag?

Reply:
When I firs tran the code with mk_geo_radcal2, I got an error that image_1.pix was not found.  I traced it down in mk_geo_radcal2 and found that the file is now called image_gamma0_1.pix.  So, I changed the name